card/lib/card/subcards.rb

```diff
@@ -228,7 +228,7 @@ def new_by_attributes name, attributes={}
     end
 
     def absolutize_subcard_name name
-      if @context_card.name =~ /^\+/
+      if @context_card.name =~ /^\+/ || name.blank?
```

Comment:
Ah, ok.  I can't think of a case other than autoname where the name should be empty by here.  (I've been a little confused all along about how exactly to handle autoname, but this seems like it fits our current approach)